### PR TITLE
Update Contribution Guidelines, add `FAQ`, `CONTACT` and `CONTRIBUTING` as separate documents, fix links

### PR DESCRIPTION
**Pull request by Julian-O**
Dear reviewers: Most of the content is moved the the RST documents, with RST-> MD edits, and other light copy-editing. Hopefully, there shouldn't be technical errors in that (apart from obsolete info).

However, there are also sections that are new (or substantially new) that you should check are factually correct as well as style, etc: 

* Supported Python versions (removed Python 2 versus 3 discussion, and rewrote)
* Relationship with kivy3, kivy4, kivy5, kivyMD
* Internationalization section

Future work:

* Add references to contact us.
* Some of the historical answers (Why Python? Relationship with PyMT) are way too long for an FAQ.

Maintainer merge checklist
* [x] Title is descriptive/clear for inclusion in release notes.
* [x] Applied a `Component: xxx` label.
* [ ] Applied the `api-deprecation` or `api-break` label.
* [ ] Applied the `release-highlight` label to be highlighted in release notes.
* [x] Added to the milestone version it was merged into.
* [ ] **Unittests** are included in PR.
* [ ] Properly documented, including `versionadded`, `versionchanged` as needed.

### CODE REVIEW

**Comment by Julian-O** [comment]
Okay, I didn't actually mean to update the Contribution Guide in the same branch, and hence into the same PR and same review as the FAQ, but... well, it is done now.

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
+# FAQ for Kivy Framework
+
+## Introduction
+[Kivy](http://kivy.org) is an open-source Python framework for developing 
+GUI apps that work cross-platform, including desktop, mobile and embedded 
+platforms, including desktop, mobile and embedded platforms.
```
Comment (misl6) [inline FAQ.md:6]: Ouch, a typo (repeated text)

File: doc/sources/contribute.rst [hunk 2]
```diff
@@ -1,64 +1,137 @@
...
-<https://www.contributor-covenant.org/version/1/4/code-of-conduct.html>`_. This
-document provides an accurate description of what is expected of you, both as a
-core developer or a first time contributor.
+sister projects is a harassment-free and positive experience for everyone. It is
+vital that all interaction is conducted in a manner conveying respect, open-mindedness and gratitude.
 
-Feedback
---------
+We have adopted V2.1 of the `Contributor Covenant Code of Conduct
+<https://www.contributor-covenant.org/version/2/1/code-of-conduct.html>`_.  Instances of abusive, harassing, or
```
Comment (misl6) [inline doc/sources/contribute.rst:119]: I got a broken link here :(

**Review by misl6 — CHANGES_REQUESTED** [review]
Hi !

Looks great, however I added a couple of comments which need to be addressed befor e merge.

**Comment by Julian-O** [comment]
Thanks for the eagle eyes. Fixes made.

**Comment by Julian-O** [comment]
Now bigger than ever before, with #8450 included.

**Comment by misl6** [comment]
@Julian-O we have a conflict 😬, can you take a look?

File: doc/README.md [hunk 3]
```diff
@@ -1,49 +1,157 @@
-Kivy - Documentation
-====================
+Kivy
```
Comment (misl6) [inline doc/README.md:1]: This README is a separate one for docs, but now it looks like a copy of the "main" one.

Is that right?

Comment (Julian-O) [reply doc/README.md]: Nope. Not right at all.

**Comment by Julian-O** [comment]
Fixed readme error. Rebased.

File: FAQ.md [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+[python-for-android](https://pypi.org/project/python-for-android/), and [kivy-ios](https://pypi.org/project/kivy-ios/),
+as appropriate.
+
+### Do you accept patches?
+
+Yes, we love patches. Obviously we don't accept every patch. Your patch has to be consistent
+with our styleguide and, more importantly, make sense. Come talk to us on Discord before you come up with bigger
+changes, especially new features.
+
+Make sure to read [our contribution guidelines](https://kivy.org/doc/stable/contribute.html).
```
Comment (misl6) [inline FAQ.md:105]: Now that will be a wrong link :)

**Review by misl6 — CHANGES_REQUESTED** [review]
Hi @Julian-O !

Great job!
I've left some comments, and some of them need to be addressed before merge.

PS: maybe we can add links to CONTACT.md, CONTRIBUTING.md, FAQ.md here?

Image 21-11-23 at 19 09

**Comment by Julian-O** [comment]
> PS: maybe we can add links to CONTACT.md, CONTRIBUTING.md, FAQ.md here?

I would love to have done that, but could not work out how.

I tried reading up on the ReadTheDocs theme, but couldn't see any parameters. Sphinx isn't my speciality, so open to suggestions

File: README.md [hunk 5]
```diff
@@ -3,29 +3,36 @@ Kivy
...
+[OpenGL ES 2.0](https://www.khronos.org/opengles/). It supports various input 
+devices and has an extensive (and extensible) widget library. With the
+same codebase, you can target Windows, OSX, Linux (including Raspberry Pi OS),
+Android, and iOS. All Kivy widgets are built with multitouch support.
 
-Kivy is MIT licensed, actively developed by a great community and is supported
-by many projects managed by the [Kivy Organization](https://kivy.org/#organization).
+Kivy is [MIT licensed](LICENSE), actively developed by a great community and is
+supported by many projects managed by the 
+[Kivy Organization](https://kivy.org/#organization).
```
[the remark this answers is not in the document]
Comment (Julian-O) [reply README.md:22]: Went with https://kivy.org/about.html instead here.

File: README.md [hunk 6]
```diff
@@ -3,29 +3,36 @@ Kivy
 
 <img align="right" height="256" src="https://raw.githubusercontent.com/kivy/kivy/master/kivy/data/logo/kivy-icon-256.png"/>
 
-Innovative user interfaces made easy.
+[Kivy](http://kivy.org) is an open-source [Python](https://python.org) framework
```
[the remark this answers is not in the document]
Comment (Julian-O) [reply README.md:6]: httpS, yes, yes, yes. Sorry.

www... okay, added throughout, but I prefer without. The "www." feels so 1990s.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,659 @@
...
+
+## Code of Conduct
+
+In the interest of fostering an open and welcoming community, we as 
+contributors and maintainers need to ensure participation in our project and 
+our sister projects is a harassment-free and positive experience for everyone. 
+It is vital that all interaction is conducted in a manner conveying respect, 
+open-mindedness and gratitude.
+
+We have adopted V2.1 of the [Contributor Covenant Code of Conduct](https://www.contributor-covenant.org/version/2/1/code-of-conduct/). Instances
```
[the remark this answers is not in the document]
Comment (Julian-O) [reply CONTRIBUTING.md:128]: Fixed this, but also moved it into the CODE_OF_CONDUCT.md file, and replaced it with a paragraph referencing the official source.

File: README.md [hunk 8]
```diff
@@ -107,7 +109,8 @@ Licenses
 
 ## Contributors
 
-This project exists thanks to all the people who contribute. [[Contribute](.github/CONTRIBUTING.md)].
+This project exists thanks to all the people who contribute.
+[[Become a contributor](.github/CONTRIBUTING.md)].
```
[the remark this answers is not in the document]
Comment (Julian-O) [reply README.md]: Agreed. Fixed

File: README.md [hunk 9]
```diff
@@ -3,29 +3,36 @@ Kivy
...
+easy.
 
-Kivy is written in Python and [Cython](http://cython.org/), based on OpenGL ES
-2, supports various input devices and has an extensive widget library. With the
-same codebase, you can target Windows, macOS, Linux, Android and iOS. All Kivy
-widgets are built with multitouch support.
+Kivy is written in Python and [Cython](https://cython.org/) and is built on
+[OpenGL ES 2.0](https://www.khronos.org/opengles/). It supports various input 
+devices and has an extensive (and extensible) widget library. With the
+same codebase, you can target Windows, OSX, Linux (including Raspberry Pi OS),
```
[the remark this answers is not in the document]
Comment (Julian-O) [reply README.md]: Done

File: CONTACT.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+[Kivy's GitHub Discussions](https://github.com/orgs/kivy/discussions) is 
+another forum where support questions are welcome, and our volunteers are 
+listening. If it takes more time to consider an answer, you will be notified 
+when you get a response.
+
+You might like to check our issue trackers out on GitHub. There is the main 
+[Kivy issue tracker](https://github.com/kivy/kivy/issues), and one for each 
+Kivy subproject. Feel free to check if others have had the same problem as you,
+ but please don't submit an issue until it is confirmed as very likely to be a 
+bug in Kivy. Ask us for help at one of the above forums before submitting an
```
[the remark this answers is not in the document]
Comment (Julian-O) [reply CONTACT.md]: Done.

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,659 @@
...
+
+  * Add some unit-tests. It can be difficult to persuade occasional
+    contributors to include sufficient unit tests. A solid bank of unit-tests
+    makes further development much faster - your small effort can have
+    long-term benefits. See 
+    [Kivy Framework Unit Tests](#kivy-framework-unit-tests) for
+    more about how our unit-tests are structured. Don't be afraid to refactor if
+    the original code is hard to test.
+
+* Kivy is extensible. You can add a new Widget or a new Python-For-Android
```
[the remark this answers is not in the document]
Comment (Julian-O) [reply CONTRIBUTING.md:87]: Good idea. Done.

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -0,0 +1,659 @@
...
+
+## Kivy Framework Unit Tests
+
+These instructions are specific to the Kivy framework (i.e. kivy/kivy in GitHub).
+
+Tests are located in the `kivy/tests` folder. If you find a bug in Kivy, a good
+thing to do is to write a minimal case showing the issue and to ask on the 
+support chnnels if the behaviour shown is intended or a real bug. If you 
+contribute your code as a 
+[unittest](http://docs.python.org/2/library/unittest.html), it will prevent the
```
[the remark this answers is not in the document]
Comment (Julian-O) [reply CONTRIBUTING.md]: Went overboard, and fixed http -> https in a number of places in the RST files.

Also, yes, linked to latest.

File: FAQ.md (hunk 4 above)
Comment (Julian-O) [reply FAQ.md]: Fixed, in separate commit

**Comment by Julian-O** [comment]
@misl6 All done. Fingers crossed.

**Review by mak8kammerer — CHANGES_REQUESTED** [review]

**Review by mak8kammerer — APPROVED** [review]

**Comment by mak8kammerer** [comment]
Sorry, there were some problems with checking the code. I closed the old reviews and opened the new ones.

**Comment by Julian-O** [comment]
7th time is the charm!

**Review by misl6 — APPROVED** [review]
🥁 LGTM. Thank you!

**Pull request merged**